crates/cargo-test-support/src/compare.rs

```diff
@@ -196,7 +196,12 @@ fn add_regex_redactions(subs: &mut snapbox::Redactions) {
         regex!(r"\.cargo/target/(?<redacted>[0-9a-f]{2}/[0-9a-f]{14})"),
     )
     .unwrap();
-    subs.insert("[HASH]", regex!(r"/[a-z0-9\-_]+-(?<redacted>[0-9a-f]{16})"))
+    subs.insert(
+        "[HASH]",
+        regex!(r"/[a-z0-9\-_]+-(?<redacted>[0-9a-f]{16})\b"),
+    )
+    .unwrap();
+    subs.insert("[HASH]", regex!(r"/pack-(?<redacted>[0-9a-f]{40})\."))
```

Comment:
This commit is last. Are tests passing in each commit and this commit is unused?

Reply:
Yes, it come with the skipped change to redact crate name and git pack filename correctly. Will dispose it.